### PR TITLE
[api] Demodulize

### PR DESCRIPTION
**Pull request by Ana06**
- Use `demodulize` Rails method instead of reimplement it by doing: `split('::').last` :bowtie:
- Use `class.name` instead `class.model_name.to_s`.
- Add `symbolified_model` method in Transport to avoid repeating code.

### CODE REVIEW

**Comment by DavidKang** [comment]
@Ana06, please, fix the first commit message 😸

File: src/api/lib/activexml/transport.rb [hunk 1]
```diff
@@ -437,5 +437,11 @@ def handle_response(http_response)
       message = http_response.to_s if message.blank?
       raise Error, message.force_encoding('UTF-8')
     end
+
+    private
+
+    def symbolified_model(model_class)
```
Comment (evanrolfe) [inline src/api/lib/activexml/transport.rb]: [Nitpick] `symbolified_model` => `symbolized_model`

Comment (bgeuken) [reply src/api/lib/activexml/transport.rb]: Nitpick²: How about symbolize_model?

Comment (Ana06) [reply src/api/lib/activexml/transport.rb]: I thought it was strange, but as it was like that everywhere 🤣

File: src/api/lib/activexml/transport.rb [hunk 2]
```diff
@@ -440,7 +440,7 @@ def handle_response(http_response)
 
     private
 
-    def symbolified_model(model_class)
+    def symbolize_model(model_class)
```
Comment (DavidKang) [inline src/api/lib/activexml/transport.rb]: This should be `symbolized_model` instead of `symbolize_model`

Comment (Ana06) [reply src/api/lib/activexml/transport.rb]: Travis also realised 🤣

**Comment by Ana06** [comment]
What happens with codecov? Can I merge this? 🤔

**Pull request merged**